package.json

```diff
@@ -70,9 +70,9 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
     "it-pair": "^1.0.0",
-    "libp2p": "^0.35.0",
-    "libp2p-floodsub": "^0.28.0",
-    "libp2p-interfaces-compliance-tests": "^2.0.3",
+    "libp2p": "libp2p/js-libp2p#feat/async-peerstore",
```

Comment:
This will need to be removed after the next libp2p release.

Longer term we need to break the circular dependencies of these components.

Reply:
We can, this was already done for pubsub as part of https://github.com/libp2p/js-libp2p/issues/857 but we need to do for the other modules. Just opened a PR to remove the dev deps: https://github.com/libp2p/js-libp2p/pull/1107